audio_reader.py

```diff
@@ -39,12 +39,19 @@ def load_vctk_audio(directory, sample_rate):
         yield audio, speaker_id
 
 
-def trim_silence(audio, threshold=0.3):
+def trim_silence(audio, filename, threshold=0.3):
```

Comment:
It was kind of nice that the `trim_silence` function  could be used on arbitrary waveforms, without supplying a filename.
We can keep the check for an empty shape, and create the warning message at the point below where `trim_silence`, then the file name wouldn't be needed here.